app/src/main/res/values/styles.xml

```diff
@@ -73,6 +73,7 @@
         <item name="drawerHeaderBackground">@color/drawerHeader_background_light</item>
         <item name="tutorialBackground">@color/tutorial_background_light</item>
         <item name="icon">@color/secondaryTextColor</item>
+        <item name="colorPrimary">@color/primaryDarkColor</item>
```

Comment:
Nor sure if related, but only on this PR several elements that should be bright are show as dark (in bright mode).

Reply:
![Screenshot_20241202-163557~2.png](https://github.com/user-attachments/assets/0e36fdc2-194b-4988-8553-b3d89927f298)

Reply:
Also:
![Screenshot_20241202-164424.png](https://github.com/user-attachments/assets/28ca2e11-9d90-477f-85d4-2e7b916b606e)

Reply:
![Screenshot_20241202-164606.png](https://github.com/user-attachments/assets/b6fccf84-9fc0-4994-8504-e66398086df6)

Reply:
Thanks for spotting it. The app literally got broken at various places. The issue appeared because of theme conflict between MaterialComponents and Appcompat so all the items which were using Appcompat got broken. Inorder, to solve it I have added `.Bridge` which allows to inherit Appcompat theme while allowing to use MaterialComponents.

Ref : https://stackoverflow.com/a/61667887
https://material.io/develop/android/docs/getting-started/#bridge-themes